### PR TITLE
feat: added external settlement endpoint

### PR DESCRIPTION
**Pull request by shashi165**
Summary:
Added provision to run settlement using an external settlement service

### CODE REVIEW

File: src/handlers/settlement-window-close.js [hunk 1]
```diff
@@ -7,32 +7,53 @@ const handler = (router, routesContext) => {
...
+                // TODO revise the implementation of the external settlement API so it sends back an
+                //  acknowledgment somehow
+                await sleep(3000);
+                ctx.response.body = await handlerHelpers.getSettlementWindows(
+                    routesContext,
+                    fromDateTime,
+                    toDateTime,
+                );
+            }
+        } else { // use the default OSS settlement service
```
Comment (elnyry-sam-k) [inline src/handlers/settlement-window-close.js:33]: @shashi165 - can you use the default as first (using ! as first if check and then have the external settlement service as else option) ?

**Review by partiallyordered — APPROVED** [review]

**Review by mdebarros — DISMISSED** [review]
Please see @elnyry-sam-k 's comment

File: src/handlers/settlement-window-close.js (hunk 1 above)
Comment (mdebarros) [reply src/handlers/settlement-window-close.js:33]: @elnyry-sam-k I assume this is to provide better backward compatibility going forward?

**Review by mdebarros — APPROVED** [review]
Approving PR for now, but inconsistencies in Settlement Endpoint configuration and backwards-compatibility support must be address in future as part of this story --> https://github.com/mojaloop/project/issues/2639

**Pull request merged**